Gemfile

```diff
@@ -37,7 +41,11 @@ platforms :jruby do
 end
 
 # Support libs
-gem "factory_bot"
+if Gem::Version.new(RUBY_VERSION) >= Gem::Version.new("3.0.0")
+  gem "factory_bot"
+else
+  gem "factory_bot", "~> 5", "< 6.4.5"
+end
```

Comment:
This is required due to the newest version of [factory_bot](https://github.com/thoughtbot/factory_bot/issues/1614) requiring Ruby 3.0.